lib/tic_tac_toe.rb

```diff
@@ -0,0 +1,64 @@
+class TicTacToe
+  attr_accessor :board
```

Comment:
- [ ] - [ ] Use attr_readers/attr_writers instead of attr_accessors. Attr_accessors allow override of class properties from outside the class. That variable can be set to anything. <TODO: explain why> ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))